packages/react-events/src/Press.js

```diff
@@ -441,11 +441,13 @@ function calculateResponderRegion(
   );
 
   const clientRect = target.getBoundingClientRect();
+  const viewportOffsetX = window.pageXOffset;
```

Comment:
Is the result still correct if the element is position fixed at the top of the viewpoint?

Reply:
I've changed this since.